jujugui/views.py

```diff
@@ -109,6 +109,8 @@ def config(request):
         # Enable/disable sandbox (demonstration) mode.
         'sandbox': sandbox_enabled,
         'sandboxSocketURL': 'wss://demo.jujucharms.com/ws',
+        # Enable/disable embedded gui
+        'embedded': settings.get('jujugui.embedded', False),
```

Comment:
Rather than doing that here, can you update options.py to set the correct defaults? That handles things like interpreting the ini providing the string "true" and makes sure we're only doing defaults in one place.